hierarc/Sampling/ParamManager/lens_param.py

```diff
@@ -143,12 +143,26 @@ def param_list(self, latex_style=False):
                     list.append(r"$\overline{\kappa}_{\rm ext}$")
                 else:
                     list.append("kappa_ext")
-            if self._kappa_ext_distribution == "GAUSSIAN":
+            if (
+                self._kappa_ext_distribution == "GAUSSIAN"
+                or self._kappa_ext_distribution == "TWO GAUSSIANS"
```

Comment:
Hi @williyamshoe I will push a PR that re-formats the LOS distribution in the form of a list of distributions (that can also be more flexible) instead, with then options in the likelihood object of each lens to select which option in the parameter space will be chosen. I hope I can push it tomorrow.

Reply:
Great, thanks!